src/Symfony/Component/Validator/Constraints/Range.php

```diff
@@ -36,14 +38,30 @@ class Range extends Constraint
     public $maxMessage = 'This value should be {{ limit }} or less.';
     public $invalidMessage = 'This value should be a valid number.';
     public $min;
+    public $minPropertyPath;
     public $max;
+    public $maxPropertyPath;
 
     public function __construct($options = null)
     {
+        if (\is_array($options)) {
+            if (isset($options['min']) && isset($options['minPropertyPath'])) {
+                throw new ConstraintDefinitionException(sprintf('The "%s" constraint requires only one of the "min" or "minPropertyPath" options to be set, not both.', \get_class($this)));
+            }
+
+            if (isset($options['max']) && isset($options['maxPropertyPath'])) {
+                throw new ConstraintDefinitionException(sprintf('The "%s" constraint requires only one of the "max" or "maxPropertyPath" options to be set, not both.', \get_class($this)));
+            }
+
+            if ((isset($options['minPropertyPath']) || isset($options['maxPropertyPath'])) && !class_exists(PropertyAccess::class)) {
+                throw new ConstraintDefinitionException(sprintf('The "%s" constraint requires the Symfony PropertyAccess component to use the "minPropertyPath" or "maxPropertyPath" option.', \get_class($this)));
```

Comment:
should be `Symfony\Component\Validator\Exception\LogicException`

Reply:
We should correct https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Constraints/AbstractComparison.php#L49 then.